### ISSUE TITLE
Redshift error

### ISSUE THREAD

**Issue by cohoe** [cohoe/workstation#103]
Unable to start GeoclueClient
GDBus.Error:org.freedesktop.DBus.Error.AccessDenied: 'redshift' 
disallowed, no agent for UID 1000.

**Comment by cohoe** [comment]
https://github.com/jonls/redshift/issues/445

**Comment by cohoe** [comment]
https://gitlab.freedesktop.org/geoclue/geoclue/-/issues/142

**Comment by cohoe** [comment]
Adding this to /etc/geoclue/geoclue.conf stopped the initial error, but it cannot actually lookup location.
```
[redshift]
allowed=true
system=false
users=1000
```

```
sudo systemctl status geoclue
● geoclue.service - Location Lookup Service
     Loaded: loaded (/usr/lib/systemd/system/geoclue.service; static)
     Active: active (running) since Wed 2021-07-14 13:57:11 EDT; 15min ago
   Main PID: 2884 (geoclue)
      Tasks: 4 (limit: 9207)
     Memory: 2.3M
     CGroup: /system.slice/geoclue.service
             └─2884 /usr/libexec/geoclue

Jul 14 13:57:11 caprica.roving.com systemd[1]: Starting Location Lookup Service...
Jul 14 13:57:11 caprica.roving.com systemd[1]: Started Location Lookup Service.
Jul 14 13:57:11 caprica.roving.com geoclue[2884]: Failed to create query: No WiFi devices available
Jul 14 13:59:32 caprica.roving.com geoclue[2884]: Failed to create query: No WiFi devices available
```

**Comment by cohoe** [comment]
Looks like the bug was introduced around early November 2020 which was in the 2.5 train. https://fedora.pkgs.org/33/fedora-x86_64/geoclue2-2.5.6-3.fc33.x86_64.rpm.html may do the trick (though as the bug report indicates, there is another problem).

**Comment by cohoe** [comment]
Confirmed. Downgrading to 2.5.6-3 and removing the special Redshift config from `/etc/geoclue/geoclue.conf` works as expected. Will have to sit on this until upstream fixes it.

**Comment by cohoe** [comment]
Looks like the fix was merged upstream, but no new release has been created. Need something >2.5.7 released after August 11th 2021 (ish).

**Comment by cohoe** [comment]
This may have been fixed in 2.5.7-6 in Fedora. https://src.fedoraproject.org/rpms/geoclue2/tree/rawhide

**Comment by cohoe** [comment]
New problem: permission denied.
